aii-ks/src/main/pan/quattor/aii/ks/config.pan

```diff
@@ -84,6 +84,10 @@ variable AII_OSINSTALL_PROTOCOL = value('/system/aii/osinstall/ks/osinstall_prot
 #
 variable AII_OSINSTALL_ROOT ?= undef;
 variable AII_OSINSTALL_OS_VERSION ?= undef;
+variable DEBUG = debug(format('%s: AII_OSINSTALL_ROOT=%s, AII_OSINSTALL_OS_VERSION=%s',
```

Comment:
with panc 10.7, the format is not necessary

Reply:
But 10.7 is not required yet I think...